frontends/concrete-python/examples/tfhers-ml/example.py

```diff
@@ -16,8 +16,26 @@
 IS_SIGNED = True
 #######################################
 
-tfhers_type = tfhers.get_type_from_params(
-    TFHERS_PARAMS_FILE,
+PARAMS_DICT = {
+    "lwe_dimension": 902,
+    "glwe_dimension": 1,
+    "polynomial_size": 4096,
+    "lwe_noise_distribution": {"Gaussian": {"std": 1.0994794733558207e-6, "mean": 0.0}},
+    "glwe_noise_distribution": {"Gaussian": {"std": 2.168404344971009e-19, "mean": 0.0}},
+    "pbs_base_log": 15,
+    "pbs_level": 2,
+    "ks_base_log": 3,
+    "ks_level": 6,
+    "message_modulus": 4,
+    "carry_modulus": 8,
+    "max_noise_level": 10,
+    "log2_p_fail": -64.084,
+    "ciphertext_modulus": {"modulus": 0, "scalar_bits": 64},
+    "encryption_key_choice": "Big",
+}
+
+tfhers_type = tfhers.get_type_from_params_dict(
+    PARAMS_DICT,
```

Comment:
We shouldn't update the example. There are explained steps to generate the file with TFHE-rs parameters in Rust. The motivation for that was to remove parameters completely from the Python file, but this is re-introducing them.

Reply:
ok, makes sense